### ISSUE TITLE
Visualizer fails to highlight active note in quantized sequences

### ISSUE THREAD

**Issue by ggilder** [magenta/magenta-js#453]
Hi, I've been playing around with magenta/music in a project, based on demos here:

https://observablehq.com/@visnup/using-magenta-music-as-a-midi-player

and here:

https://hello-magenta.glitch.me/

I've observed that if I use a `PianoRollCanvasVisualizer` or `PianoRollSvgVisualizer` with a quantized sequence, it will highlight the first note in the sequence, but fail to highlight subsequent active notes.

In my own project, I've been able to work around this by first unquantizing the sequence:

```
let seq = mm.sequences.unquantizeSequence(seq);
```

Then the visualizer works as expected.

cc @visnup

**Comment by notwaldorf** [comment]
You’re totally right! I remember seeing this, and I think it’s actually a breaking change I accidentally introduced in #375 (because the player automatically converts it to a quantized sequence, and now it can’t find the correct svg rect based on the active note callback)

I’ll take a look at fixing this, or at least documenting the problem.

**Comment by cdfmlr** [comment]
Hi, I am wondering if there's any progress on the issue?
